doc/whats-new.rst

```diff
@@ -27,6 +27,12 @@ Breaking changes
   (:pull:`3274`)
   By `Elliott Sales de Andrade <https://github.com/QuLogic>`_
 
+- New deprecations (behavior will be changed in xarray 0.17):
+  - ``dim`` argument to :py:meth:`DataArray.integrate` is being deprecated in
```

Comment:
I think you need a newline before the nested list (this should be the reason why the docs CI fails)